packages/smart/tasks/setTrustedResolution.ts

```diff
@@ -0,0 +1,22 @@
+import { task, types } from "hardhat/config";
+
+import "hardhat/types/config";
+import { buildContractInterfaces, ContractInterfaces } from "..";
+
+task("setTrustedResolution", "Set turbo resolution for the TrustedArbiter")
+  .addParam("turbo", undefined, undefined, types.int)
+  .addParam("outcomes", undefined, undefined, types.json)
+  .setAction(async ({ turbo, outcomes }, hre) => {
+    if (typeof turbo !== "number") return;
```

Comment:
Is this possible? I think hardhat will fail before you get here since you set the type to types.int.